### PR TITLE
Support for make container-uninstall

### PR DESCRIPTION
**Pull request by niks3089**
I made the following changes when you run `make container-uninstall`:

1. Delete `runnc-build` image from docker
2. Do a make clean on runnc
3. Remove the runnc and nabla-run binaries from the system.

---
This change is [Reviewable](https://reviewable.io/reviews/nabla-containers/runnc/67)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 vendor/
 bin/
 build/
+*nabla
```
Comment (lumjjb) [inline .gitignore]: Is this for just `.nabla` files? Should this be `*.nabla` if that's the case?

File: Makefile [hunk 2]
```diff
@@ -48,6 +48,11 @@ container-install:
 	sudo docker build . -f Dockerfile.build -t runnc-build
 	sudo docker run --rm -v /opt/runnc/:/opt/runnc/ -v /usr/local/bin:/usr/local/bin -v ${PWD}:/go/src/github.com/nabla-containers/runnc -w /go/src/github.com/nabla-containers/runnc runnc-build make install
 
+container-uninstall:
+	sudo docker rmi runnc-build | true
```
Comment (lumjjb) [inline Makefile]: I think `sudo docker rmi -f` works here.

File: .gitignore (hunk 1 above)
Comment (niks3089) [reply .gitignore]: Ya, that's right

File: Makefile (hunk 2 above)
Comment (niks3089) [reply Makefile]: Yes, it does.

**Pull request merged**